### PR TITLE
Code Issue in /plugins/notifications/sendgrid #7068

### PR DESCRIPTION
**Pull request by tesfamickal**
changed order_shipped_template to be renamed to order_shipment_created_template

### CODE REVIEW

**Comment by Mohmdev** [comment]
> changed order_shipped_template to be renamed to order_shipment_created_template

Yes I found this issue too. Took me a day to figure this out.

File: www/apps/docs/content/plugins/notifications/sendgrid.mdx [hunk 1]
```diff
@@ -686,7 +686,7 @@ You don’t have to create a template for every type in the reference. You can s
 
 ### Order Shipment Created
 
-**Key in plugin options:** `order_shipped_template`
+**Key in plugin options:** `order_shipment_created_template`
```
Comment (shahednasser) [inline www/apps/docs/content/plugins/notifications/sendgrid.mdx:689]:
```suggestion
**Key in plugin options:** `order_shipment_created_template` in the root options, and `order_shipped_template` in the localization options.
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
@tesfamickal can I get you to rebase to the `v1.x` branch?

File: www/apps/docs/content/plugins/notifications/sendgrid.mdx (hunk 1 above)
Comment (tesfamickal) [reply www/apps/docs/content/plugins/notifications/sendgrid.mdx:689]: @shahednasser sure thing

**Comment by riqwan** [comment]
Hey, thanks for the PR! Since v2 brought a lot of architectural changes, we will be closing this PR since it no longer applies to our new setup.

If you would still like to open this PR, please rebase with the v1 branch and re-request a review & we'll get back on it quickly.

**Pull request closed without merging**